Minesweeper.py

```diff
@@ -20,34 +20,42 @@ class GameState(object):
     """ changes the game states which are used in the code to fulfill conditions"""
     def __init__(self):
         """sets the states to their beginning values, which can then be chaged later"""
+        
+        # sets the beginning state of tutorial
         self.tutorial = True
+
+        # sets the beginning state of game
         self.game = False
+
+        # sets the beginning state of game_over
         self.game_over = False
+
+        # sets the beginning state of game_won
         self.game_won = False
 
     def set_state_tutorial(self):
-        if self.tutorial == True:
-            self.tutorial = False
-        elif self.tutorial == False:
-            self.tutorial = True
+        self.tutorial = True
+        self.game_over = False
+        self.game_won = False
+        self.game = False
 
     def set_state_game(self):
-        if self.game == True:
-            self.game = False
-        elif self.game == False:
-            self.game = True
+        self.tutorial = False
+        self.game_over = False
+        self.game_won = False
+        self.game = True
 
     def set_state_game_over(self):
-        if self.game_over == True:
-            self.game_over = False
-        elif self.game_over == False:
-            self.game_over = True
+        self.tutorial = False
+        self.game_over = True
+        self.game_won = False
+        self.game = False
     
     def set_state_game_won(self):
-        if self.game_won == True:
-            self.game_won = False
-        elif self.game_won == False:
-            self.game_won = True
+        self.tutorial = False
+        self.game_over = False
+        self.game_won = True
+        self.game = False
```

Comment:
Better :). Could we now reduce the copy-paste feeling and increase readability, by creating a helper function, e.g. def clear_state(self): that sets all the values to False, and then have all of these code sections call it first, and then only set one state value to True?